index.js

```diff
@@ -4,15 +4,16 @@ const moment = require("moment");
 
 const REPO = "rotabull";
 const OWNER = "rotabull";
-const CLUBHOUSE_BASE_URL = "https://app.clubhouse.io/rotabull/story/";
+const CLUBHOUSE_BASE_URL = "https://app.shortcut.com/rotabull/story/";
 const HEROKU_API_BASE_URL = "https://api.heroku.com";
 const GITHUB_API_BASE_URL = "https://api.github.com";
-const CLUBHOUSE_API_BASE_URL = "https://api.clubhouse.io/api/v3";
+const CLUBHOUSE_API_BASE_URL = "https://api.shortcut.com/api/v3";
```

Comment:
looking at the doc https://shortcut.com/api/rest/v3#We-changed-our-name, looks like the base url is 
**https://api.app.shortcut.com/api/v3/**

Reply:
fixed https://github.com/rotabull/rotabull-release-helper/pull/20/commits/03f6d2e52f51cf62493f3590d13ad7787c4f3ccb